docs_user/assemblies/assembly_prepare_director_operator_and_rhoso_for_adoption_process.adoc

```diff
@@ -0,0 +1,190 @@
+ifdef::context[:parent-context: {context}]
+
+[id="prepare_director_operator_and_rhoso_for_adoption_process"]
+
+:context: migrating-databases
+
+= Preparing director operator for adoption
+
```

Comment:
We need an introduction that tells the customer what they're about to do in the procedure. I propose something like:
"Before you adopt your Red Hat OpenStack Platform (RHOSP) director Operator (OSPdO) deployment, you must prepare the Controller nodes for adoption to the RHOSO environment."

^ I think there's more to it than that. Please feel free to add suggestions.

Reply:
Katie, the controller preperation is for customers with a specific setup - the one listed in the below note (shared OCP environment) 
Would it make more sense to put this introduction under the below NOTE section ?

Reply:
currently put it on line 14

Reply:
The intro that describes what customers are doing in the procedure should go before the note. Typically we don't start an intro with a note.

Reply:
ok , switching the order